tests/codeutils/test_identifiersplitting.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import dpu_utils.codeutils.identifiersplitting as split
+
+class TestSplitCamelCase(unittest.TestCase):
```

Comment:
Could you rename this to `TestIdentifierSplitting`? (the test actually includes cases of snake_case)

Reply:
This class actually only tests the `split_camelcase()` function. The `snake_case` test was just testing the case of splitting out underscores (i.e. `snake_case` -> `["snake", "_", "case"]`), but I think that idea is sufficiently covered by `test_special_characters_are_split()`, so I deleted the `snake_case` test.

I was planning on making another class in a separate PR for testing `split_identifier_into_parts()` since it handles snake case and uppercase characters differently than `split_camelcase()`. Would you prefer if that was grouped into the same class / do you still want me to rename the class?